### DISCUSSION TITLE
Uncaught TypeError: Cannot convert a BigInt value to a number

### DISCUSSION THREAD

**Discussion by Bhushanlal** [paulmillr/noble-curves#94] [Q&A]
Hello,
I am Abhishek and i am working on a project which is based on the react and blockchain.In this project @nobel/curve is used. Due to this on the live server it is creating issue. (have a look to attached screenshot).
![image](https://github.com/paulmillr/noble-curves/assets/33514064/23ce2fb7-7d6f-407d-b617-38bde587ac15)
After research on this issue many other person also face this errors but i unable to get the solution. if you know something about it please share with me to solve this issue.

**Comment by paulmillr** [comment]
https://github.com/paulmillr/noble-curves/issues/43#issuecomment-1537129629
